lib/saml_authenticator.rb

```diff
@@ -121,6 +120,7 @@ def after_authenticate(auth)
       auth.info[:nickname] = uid.to_s
     end
 
+    auth.extra = { "raw_info" => attributes.attributes }
```

Comment:
I'm wondering if there's a corresponding change in https://github.com/discourse/discourse-saml/blob/main/spec/saml_authenticator_spec.rb

Reply:
There isn't - the omniauth strategy is mocked in tests, so this commit essentially updates the production behavior to match what we currently expect in tests.

I intend to add a proper integration spec for SAML, including the omniauth strategy. But I'd like to get this fix merged ASAP since it's causing issues in production.